pelorus-operator/Makefile

```diff
@@ -146,20 +150,36 @@ ifeq (,$(shell which helm-operator 2>/dev/null))
 	@{ \
 	set -e ;\
 	mkdir -p $(dir $(HELM_OPERATOR)) ;\
-	curl -sSLo $(HELM_OPERATOR) https://github.com/operator-framework/operator-sdk/releases/download/v1.28.1/helm-operator_$(OS)_$(ARCH) ;\
+	curl -sSLo $(HELM_OPERATOR) https://github.com/operator-framework/operator-sdk/releases/download/v1.30.0/helm-operator_$(OS)_$(ARCH) ;\
 	chmod +x $(HELM_OPERATOR) ;\
 	}
 else
 HELM_OPERATOR = $(shell which helm-operator)
 endif
 endif
 
+.PHONY: operator-sdk
```

Comment:
these changes (below this line) I think do not make sense for the project, at least at the moment

we use `.venv` folder and `scripts/install_dev_tools.sh` for tools download, we should change the script not add commands here

Reply:
I think would be better to avoid changing CI related things in this PR

Reply:
re-generated using operator-sdk 1.28.1, so no changes here.